### PR TITLE
S3 methods for _df functions

### PR DESCRIPTION
**Pull request by dpprdan**
This PR adds S3 methods to `oc_forward_df` and `oc_reverse_df` so it is now possible to pass character (forward) or numeric vectors (reverse) to them:

``` r
library(opencage)
oc_forward_df(c("Los Angeles City Hall", "Notre Dame, Paris", "Elbphilharmonie"))
#> # A tibble: 3 x 4
#>   placename       oc_lat  oc_lng oc_formatted                              
#>   <chr>            <dbl>   <dbl> <chr>                                     
#> 1 Los Angeles Ci~   34.1 -118.   Los Angeles City Hall, 200 North Spring S~
#> 2 Notre Dame, Pa~   48.9    2.35 Cathedral of Notre Dame, 6 Parvis Notre-D~
#> 3 Elbphilharmonie   53.5    9.98 Elbe Philharmonic Hall, Platz der Deutsch~

oc_reverse_df(
  lat = c(40.7127281, -22.9110137, 35.6828387),
  lon = c(-74.0060152, -43.2093727, 139.7594549)
)
#> # A tibble: 3 x 3
#>   latitude longitude oc_formatted                                          
#>      <dbl>     <dbl> <chr>                                                 
#> 1     40.7     -74.0 New York City Hall, 260 Broadway, New York, NY 10000,~
#> 2    -22.9     -43.2 Viaduto dos Fuzileiros, Rio de Janeiro - RJ, 20270-15~
#> 3     35.7     140.  Kokyo Gaien National Garden, Chiyoda, Tokyo 100-0002,~
```

As you can see, the vectors are bound to the resulting tibble. However, this is only the case for `placename`, and `latitude`/`longitude`, respectively. I guess, I did not see the need for passing e.g. a `language` vector on as well?

The motivation for this is that I found `oc_forward("Hamburg") %>% bind_cols()` a bit cumbersome. In addition, we already [discussed](https://github.com/ropensci/opencage/pull/42#issuecomment-367320323) allowing (character) vectors as input to the \_df functions briefly before.

### CODE REVIEW

File: R/oc_forward.R [hunk 1]
```diff
@@ -340,15 +340,29 @@ oc_forward <-
 #' oc_forward_df(df2, placename = locations,
 #'               countrycode = countrycode)
 #' }
+oc_forward_df <- function(...) UseMethod("oc_forward_df")
 
-oc_forward_df <-
+# No @name so it does not show up in the docs.
```
Comment (maelle) [inline R/oc_forward.R]: Why not `@NoRd`?

**Review by maelle — APPROVED** [review]

**Comment by maelle** [comment]
Thank you!

File: R/oc_forward.R (hunk 1 above)
Comment (dpprdan) [reply R/oc_forward.R]: Dunno. Not specifiying `@name` is sufficient for it to not show up in docs, so `@NoRd` is redudant in a way, but I agree that it conveys intent more succinctly.

Comment (maelle) [reply R/oc_forward.R]: as you prefer!

Comment (dpprdan) [reply R/oc_forward.R]: Already made the change.

Comment (dpprdan) [reply R/oc_forward.R]: (and I thought, I had already merged, but apparently not)

**Pull request merged**